android/assets/jsons/translations/Korean.properties

```diff
@@ -266,20 +264,16 @@ Demanding a Worker from small City-State =
 Very recently paid tribute = 
  # Requires translation!
 Recently paid tribute = 
- # Requires translation!
-Influence below -30 = 
- # Requires translation!
-Military Rank = 
- # Requires translation!
-Military near City-State = 
```

Comment:
It isn't necessary to remove all of these `# Requires translation!`, as that is done automatically whenever a new version is released

Reply:
Just a head's up to save you some work :)

Reply:
I appreciate again. I know it all goes(or comes if i delete translation) automatically, but it's my style.
I'm not even amateur(?) for translation. I'm just ordinary Korean speaker, and i'm working only on Github online.
I use "# req" with Ctrl + F a lot to jump where i have to work.
If you don't mind, i'll just keep erase # requires translation lines.